ospo-book/chapters/02-chapter.md

```diff
@@ -1 +1,107 @@
-TBD
\ No newline at end of file
+# Chapter 2: Understanding the Value of Open Source Program Offices
+
+- [Introduction](#introduction)
+    - Reasons Behind Starting an OSPO
+    - Reasons Behind Sustaining an OSPO
+- [Assessing Value of Open Source Activity](#assessing-value-of-open-source-activity) - `✅ Assessment`
+- [Antipatterns](#antipatterns) - `🚫 OSPO Antipatterns`
+- [Resources](#resources) - `📚 Continue Here`
+
+# Introduction
+
+From end-user companies and software companies to universities and public administrations, an organization's relationship with open source and the value it can get from an OSPO depend on the type of organization it is. However, we can still identify some critical problems that arise when getting involved in open source on an organizational level, which motivates the creation and continuity of an OSPO.
```

Comment:
I would say that the relationship with open source and the value it can get from an OSPO not only depend on the type of organisatie, but differs for each organisation. 

```suggestion
From end-user companies and software companies to universities and public administrations, an organization's relationship with open source and the value it can get from an OSPO is different per organization. However, we can still identify some critical problems that arise when getting involved in open source on an organizational level, which motivates the creation and continuity of an OSPO.
```